src/Agent.Listener/Capabilities/AgentCapabilitiesProvider.cs

```diff
@@ -34,6 +35,10 @@ public Task<List<Capability>> GetCapabilitiesAsync(AgentSettings settings, Cance
 #if OS_WINDOWS
             Add(capabilities, "Agent.OSVersion", GetOSVersionString());
             Add(capabilities, "Cmd", Environment.GetEnvironmentVariable("comspec"));
+
+            //This capability gets set irrespective of agent being run in auto-logon mode or not.
+            var configurationManager = HostContext.GetService<IConfigurationManager>();
+            Add(capabilities, "InteractiveSession", (!configurationManager.IsServiceConfigured()).ToString());
```

Comment:
I'll close with Prachi on this.

---
In reply to: [117337154](https://github.com/Microsoft/vsts-agent/pull/978#discussion_r117337154) [](ancestors = 117337154)